linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/HelpPageHandler.scala

```diff
@@ -0,0 +1,19 @@
+package io.buoyant.linkerd.admin
+
+import com.twitter.finagle.Service
+import com.twitter.finagle.http.{Request, Response, Status}
+import com.twitter.util.Future
+
+private[admin] class HelpPageHandler extends Service[Request, Response] {
+
+  override def apply(req: Request): Future[Response] = AdminHandler.mkResponse(html)
+
+  def html = {
+    AdminHandler.html(
+      content = s"""
+        <div class="container main help-page"></div>
```

Comment:
I had the same question as @adleong. Given that this page is 100% static, no template variable needed, I think it should be hardcoded here. (I know it's a pain to develop that way, but the help page won't need to change too often)